### PR TITLE
Cypress chores

### PR DESCRIPTION
**Pull request by webshuriken**
### Chores page

- You can visit the chores page
- Create new chores list
- Cancel edit of chores list name action
- Edit the chores list name
- Cancel delete action of chores list
- Delete chores list

### Chores Items page

- Navigate to chores list, matching URL and page title
- Creates two items
- Checks items as completed
- Unchecked items, as completed
- Cancel delete item action
- Deletes item

### CODE REVIEW

File: cypress/e2e/chores-items.cy.ts
Comment (AliMcG) [inline cypress/e2e/chores-items.cy.ts:1]: Great use of assertions. I did not know about `.parents()`, very interesting.

**Review by AliMcG — COMMENTED** [review]
The tests run and the code looks good. Great use of the cypress best practices (data-cy=""some-selector").
Only one comment to address.

File: cypress/e2e/chores.cy.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+ *  - click on the "Delete" button inside the modal
+ */
+
+describe('Test suite for the Chores route', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  afterEach(() => {
+    // cy.logout();
```
[the remark this answers is not in the document]
Comment (webshuriken) [reply cypress/e2e/chores.cy.ts:33]: Yes it is. Nice catch.

**Comment by webshuriken** [comment]
@AliMcG changes made and ready

**Review by AliMcG — APPROVED** [review]

**Pull request merged**